pyproject.toml

```diff
@@ -24,6 +24,7 @@ dependencies = [
     "ndfilters==0.3.0",
     "colorsynth==0.1.5",
     "regridding==0.1.1",
+    "xarray==2025.3.1",
```

Comment:
I think you should unpin the version of `xarray`, it's my thinking we only use strict version numbers for our own deps since they're tightly coupled.